lib/utilities/dot_dict.py

```diff
@@ -17,3 +17,27 @@ def __init__(self, dct={}):
             if hasattr(value, 'keys'):
                 value = DotDict(value)
             self[key] = value
+
+    def get(self, key):
```

Comment:
This class is somewhat unit tested, so is there any chance we can add a few tests for this function to https://github.com/mozilla/MozDef/blob/master/tests/lib/utilities/test_dot_dict.py

Reply:
Sure, added some that should exercise the get('string.string') a bit